elastic/apm-server
# Update to elastic/beats@18f4e405ad11 (7.x)

## Motivation/summary

Update beats to elastic/beats@18f4e405ad11, bringing in https://github.com/elastic/beats/pull/25020

## How to test these changes

1. Install apm-server (don't just make apm-server, this is a shortcut for development and does not build PIE)
2. Check the binary is PIE (see https://unix.stackexchange.com/questions/89211/how-to-test-whether-a-linux-binary-was-compiled-as-position-independent-code)

## Related issues

#4960
